src/backend/src/services/design-reviews.services.ts

```diff
@@ -124,10 +119,13 @@ export default class DesignReviewsService {
       throw new HttpException(400, 'Design review cannot be scheduled for a past day');
     }
 
+    const futureDay = new Date();
+    futureDay.setDate(futureDay.getDate() + 1);
+
     const designReview = await prisma.design_Review.create({
       data: {
         dateScheduled,
-        dateCreated: new Date(),
+        dateCreated: futureDay,
```

Comment:
this is supposed to be new Date(), cause we want it to be the day that the design review was created

Reply:
sometimes the test fail and say it has to be a future date so i thought because of the date issue you guys were talking about before this would complelty fix that

Reply:
that test is talking about dateScheduled, not dateCreated. I just pushed code to fix the issue